### DISCUSSION TITLE
Pocketbase Alternative For .htaccess

### DISCUSSION THREAD

**Discussion by Gursukh** [pocketbase/pocketbase#1694] [Q&A] [answered]
I'm serving a static page from my pb_public. From the index, I want to redirect to another page, "/page" without having to put the ".html". I tried using a ".htaccess" file within the pb_public file but it seems to have no effect. What could I use to accomplish this?

**Comment by ganigeorgiev** [comment, marked as answer]
You can try creating a directory `page` with `index.html` file inside it:

```
/page
   - index.html
```

If you are using a frontend framework like SvelteKit, you can set [`export const trailingSlash = 'always'`](https://kit.svelte.dev/docs/page-options#trailingslash).
